components/usage/pkg/stripe/webhook.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package stripe
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/gitpod-io/gitpod/common-go/log"
+	"github.com/stripe/stripe-go/v72"
+)
+
+type WebhookHandler struct{}
+
+func NewWebhookHandler() *WebhookHandler {
+	return &WebhookHandler{}
+}
+
+func (h *WebhookHandler) Handle(w http.ResponseWriter, req *http.Request) {
+	const maxBodyBytes = int64(65536)
```

Comment:
I believe there' a middleware for this

Reply:
I couldn't find any middleware that does exactly this. Did you have something in mind?